frontend/src/environments/environment.trees.ts

```diff
@@ -0,0 +1,6 @@
+export const environment = {
+  production: true,
+  apiUrl: 'https://fs-intake-api-trees.app.cloud.gov/',
+  buildDate: 'Mon Jun 12 2017 22:17:33 GMT-0500 (CDT)',
```

Comment:
Do the build dates & versions actually get populated on the build?

Reply:
updated this to use `new Date()`.

Reply:
@lauraGgit replace.build.js script currently replaces date to the current date during the deployment. However, that works on environment.`prod` with intake but not on `trees` deployment. Let us discuss or update this once intake is complete.